.github/stale.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 60
+
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 14
+
+# Issues or Pull Requests with these labels will never be considered stale
+exemptLabels:
+  - "enhancement"
+
+# Label to use when marking as stale
+staleLabel: "awaiting input"
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
```

Comment:
"This issue has been marked as stale because it has not had any recent activity. If no further activity occurs, it will be closed. Please feel free to open a new issue / Please feel free to reopen this issue if needed."

At least the "automatically" seems redundant. Also, for issues, "Thank you for you contributions" seems strange.